lib/azurex/blob.ex

```diff
@@ -170,21 +189,22 @@ defmodule Azurex.Blob do
   @doc """
   Copies a blob to a destination.
   """
-  @spec copy_blob(String.t(), String.t(), optional_string) ::
+  @spec copy_blob(String.t(), String.t(), Config.config_overrides()) ::
           {:ok, HTTPoison.Response.t()} | {:error, term()}
-  def copy_blob(source_name, destination_name, container \\ nil) do
+  def copy_blob(source_name, destination_name, overrides \\ []) do
```

Comment:
I suggest documenting that the overrides apply to both source and destination, so you can't use this method to copy between accounts.

Reply:
I have added a line in the doc mentioning it